python/paddle/fluid/framework.py

```diff
@@ -1817,6 +1817,89 @@ def _sliceAndConcatVar(self, item, axis):
     def __getitem__(self, item):
         return _getitem_impl_(self, item)
 
+    def __setitem__(self, item, value):
+        inputs = {'Input': self}
+
+        # 1. Parse item
+        if not isinstance(item, tuple):
+            item = [item]
+
+        axes = []
+        starts = []
+        ends = []
+        max_integer = 2147483647
```

Comment:
In Python, you can get max int by sys.maxint and the value will be different across different platforms. I saw you were using max_integer for `end`, does the different value makes something different? When I read your code, I feel it won't be different but I would like to comment to let you make sure.

Reply:
Thanks, use `sys.maxsize` to replace it

<!--Here, 2147483647 is the max value of c++ `int` on 32 bit operating system.-->